spec/add_git_remote_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe Fastlane::Actions::AddGitRemoteAction do
+  describe 'Add git remote' do
+    it 'should call git remote name path ' do
+      expect(Fastlane::Actions).to receive(:sh).with("git remote add testRemote https://github.com/meniga/mobile-sdk-ios")
```

Comment:
Can `expect` be called after `Fastlane::Actions::AddGitRemoteAction.run`?

Reply:
Expectation connected with messages need to be before run. https://relishapp.com/rspec/rspec-mocks/docs/basics/expecting-messages